std/assembly/string.ts

```diff
@@ -55,7 +53,7 @@ import { idof } from "./builtins";
     if (<u32>pos >= <u32>len) return -1; // (undefined)
     var first = <i32>load<u16>(changetype<usize>(this) + (<usize>pos << 1));
     if ((first & 0xFC00) != 0xD800 || pos + 1 == len) return first;
-    var second = <i32>load<u16>(changetype<usize>(this) + ((<usize>pos + 1) << 1));
+    var second = <i32>load<u16>(changetype<usize>(this) + (<usize>pos << 1), 2);
```

Comment:
Not sure if worth, but this can reuse `changetype<usize>(this) + (<usize>pos << 1)` from `first`

Reply:
It usually reduce size but hurt performance because `load(ptr + idx * 4)` usually lowered to `mov edx, dword ptr [eax + ecx * 4 + #immCont]` which is one instruction. Otherwise it will be 3-4 instructions